app/assets/stylesheets/index.css

```diff
@@ -0,0 +1,62 @@
+.photo {
+  height: 600px;
+  width: 800px;
+  border-radius: 10px;
+}
+.image-wrapper {
+  width: 80%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  border-radius: 10px;
+  padding-top: 10px;
+}
+.image-overlay-content {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  overflow: hidden;
+  top: 0;
+  left: 0; }
+.overlay-fade-in p {
+  letter-spacing: 0.15em;
+  color: #f4f4f4;
+  font-size: 24px;
+  opacity: 0;
+  transition: all 0.2s linear; }
+.overlay-fade-in img {
+  transition: all 0.2s linear; }
+.overlay-fade-in .image-overlay-content {
+  opacity: 0;
+  transition: all 0.4s ease-in-out; }
+.overlay-fade-in h2 {
+  color: #f4f4f4;
+  font-size: 38px;
+  margin-top: 40%;
+  opacity: 0;
+  background: rgba(0, 0, 0, 0);
+  transition: all 0.2s ease-in-out; }
+.overlay-fade-in .button {
+  display: inline-block;
+  text-decoration: none;
+  padding: 7px 14px;
+  background: #FFF;
+  color: #f4f4f4;
+  text-transform: uppercase;
+  box-shadow: 0 0 1px #000;
+  position: relative;
+  opacity: 0;
+  transition: all 0.2s ease-in-out; }
+.overlay-fade-in .button:hover {
+  box-shadow: 0 0 5px #000; }
+.overlay-fade-in:hover img {
+  transform: scale(1.2); }
+.overlay-fade-in:hover .image-overlay-content {
+  opacity: 1; }
+.overlay-fade-in:hover h2, .overlay-fade-in p, .overlay-fade-in .button {
+  opacity: 1; }
+.overlay-fade-in:hover p {
+  transition-delay: 0.1s; }
+.overlay-fade-in:hover .button {
+  transition-delay: 0.2s; }
```

Comment:
Add a new line after each class selector styling similar to that in  `foundation_and_overrides.scss`.

For example 59 should now be 
`.overlay-fade-in:hover p {
  transition-delay: 0.1s; 
}
#empty line here to separate the next class styling`

Reply:
The closing `}` should be brought down to the next line
Currently line 71 is 

`.overlay-fade-in:hover p {
    transition-delay: 0.1s; }`

But I would like 
`.overlay-fade-in:hover p {
    transition-delay: 0.1s; 
 }`